src/main/java/org/somerville/swag/data/source/TextFileWriter.java

```diff
@@ -0,0 +1,29 @@
+package org.somerville.swag.data.source;
+
+import org.somerville.swag.data.exeption.FileWriterException;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class TextFileWriter {
```

Comment:
Is the only file we are writing to the log file? or are there more file we plan to write to?

Reply:
I've made sure to write to an interface instead of a single implementation, this code gives us the freedom to create other types of FileWriters in the future if need be (CSV, JSON etc)